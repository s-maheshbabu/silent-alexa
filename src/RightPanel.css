/** Scaffolding **/
#rightpanel {
  display: grid;
  grid-template-rows: auto;
  height: calc(100vh - 60px);
}

.panel-body {
  overflow: auto;
}