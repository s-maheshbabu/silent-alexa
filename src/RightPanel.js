import React from "react";

export default function RightPanel() {
  return (
    <div id="rightpanel">
      <div className="panel-body">Right side content</div>
    </div>
  );
}