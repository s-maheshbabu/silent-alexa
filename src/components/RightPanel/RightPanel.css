/** Scaffolding **/
#rightpanel {
  display: grid;
  grid-template-rows: auto;
  height: calc(100vh - 90px);
}

.panel-body {
  overflow: auto;
}
