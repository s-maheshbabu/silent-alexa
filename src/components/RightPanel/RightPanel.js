import React from "react";
import "./RightPanel.css";

export default function RightPanel() {
  return (
    <div id="rightpanel">
      <div className="panel-body">Right side content</div>
    </div>
  );
}
