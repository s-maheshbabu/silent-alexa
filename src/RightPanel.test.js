import React from "react";
import {mount} from "enzyme";
import RightPanel from "./RightPanel";

it("renders RightPanel without crashing", () => {
  mount(<RightPanel />);
});

it("renders correctly (snapshot testing)", () => {
  const wrapper = mount(<RightPanel />);
  expect(wrapper).toMatchSnapshot();

  wrapper.unmount();
});