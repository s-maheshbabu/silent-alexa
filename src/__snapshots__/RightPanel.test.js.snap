// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders RightPanel without crashing 1`] = `
<div
  id="rightpanel"
>
  <div
    className="panel-body"
  >
    Right side content
  </div>
</div>
`;
